build.sbt

```diff
@@ -56,20 +57,23 @@ lazy val common = project.in(file("faunadb-common"))
     javacOptions ++= Seq("-source", "1.7", "-target", "1.7"),
     apiURL := Some(url(s"http://faunadb.github.io/faunadb-jvm/$driverVersion/faunadb-common/api/")),
 
-    (javacOptions in doc) := Seq("-source", "1.7", "-Xdoclint:none",
+    (javacOptions in doc) := Seq("-source", "1.7",
       "-link", "http://docs.oracle.com/javase/7/docs/api/",
       "-link", s"http://google.github.io/guava/releases/$guavaVersion/api/docs/",
       "-link", s"http://fasterxml.github.io/jackson-databind/javadoc/$jacksonDocVersion/",
       "-link", s"http://dropwizard.github.io/metrics/$metricsVersion/apidocs/",
-      "-linkoffline", s"http://static.javadoc.io/org.asynchttpclient/async-http-client/$asyncHttpClientVersion/", s"./faunadb-common/doc/org.asynchttpclient/async-http-client/$asyncHttpClientVersion/"),
+      "-linkoffline", s"http://static.javadoc.io/com.ning/async-http-client/$asyncHttpClientVersion/", s"./faunadb-common/doc/com.ning/async-http-client/$asyncHttpClientVersion/"
```

Comment:
I got you. In this case we don't need it anymore because the version 1.9.39 provides this file http://static.javadoc.io/com.ning/async-http-client/1.9.39/package-list